py-usfm-parser/tests/test_json_conversion.py

```diff
@@ -116,3 +116,54 @@ def test_output_is_valid_usj(file_path):
     assert not test_parser.errors, test_parser.errors
     usj_dict = test_parser.to_usj()
     validate(instance=usj_dict, schema=USJ_SCHEMA)
+
+def remove_newlines_in_text(usj_dict):
+    '''The test samples in testsuite do not preserve new lines in. But we do in usfm-grammar.
+    So removing them just for comparison'''
+    if "content" in usj_dict:
+        for i,item in enumerate(usj_dict["content"]):
+            if isinstance(item, str):
+                usj_dict['content'][i] = item.replace("\n", " ")
+                usj_dict['content'][i] = usj_dict['content'][i].replace("  ", " ")
+                usj_dict['content'][i] = usj_dict['content'][i].replace("     ", " ")
+                continue
+            if "sid" in item and "PSA 09" in item['sid']: # for /usfmjsTests/tstudio/origin.usfm
+                item['sid'] = item['sid'].replace("PSA 091:01", "PSA 91:1")
```

Comment:
Are we stripping away the `0` in the front?

Reply:
We are not, in usfm-grammar. But the USXs in testsuite have them stripped in some places, but not all. I am just changing them in test, to get the comparison to pass, as this is known discrepancy. Same thing as with the space handling. That's what is going on here